<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="rightVLayout">
        <item>
         <layout class="QHBoxLayout" name="statHLayout">
          <item>
           <layout class="QVBoxLayout" name="episodeStatVLayout">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Episode stats:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="frameAmountLabel">
              <property name="text">
               <string>Frames: -</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="startEpisodeLabel">
              <property name="text">
               <string>Start: -</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="endEpisodeLabel">
              <property name="text">
               <string>End: -</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lengthEpisodeLabel">
              <property name="text">
               <string>Length: -</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="sensorEpisodeLabel">
              <property name="text">
               <string>Sensors: -</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="frameStatVLayout">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Frame stats:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="minLabel">
              <property name="text">
               <string>min: -</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="maxLabel">
              <property name="text">
               <string>max: -</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="avLabel">
              <property name="text">
               <string>av: -</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="frameTimeLabel">
              <property name="text">
               <string>Fame time: -</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="sensorLabel">
              <property name="text">
               <string>Sensor: </string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="backwardMoreButton">
            <property name="text">
             <string>&lt;&lt;&lt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="backwardOneButton">
            <property name="text">
             <string>&lt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="forwardOneButton">
            <property name="text">
             <string>&gt; </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="forwardMoreButton">
            <property name="text">
             <string>&gt;&gt;&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSlider" name="timeSlider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="csvHLayout"/>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="startLabel">
          <property name="text">
           <string>Start time</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QDateTimeEdit" name="startTimeEdit">
            <property name="displayFormat">
             <string>dd/MM/yyyy HH:mm:ss</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ignoreStartCheckbox">
            <property name="text">
             <string/>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="stopLabel">
          <property name="text">
           <string>Stop time</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QDateTimeEdit" name="stopTimeEdit">
            <property name="displayFormat">
             <string>dd/MM/yyyy HH:mm:ss</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ignoreStopCheckbox">
            <property name="text">
             <string/>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="connectLabel">
          <property name="toolTip">
           <string extracomment="Zero to always connect"/>
          </property>
          <property name="text">
           <string>Connect time (seconds)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="connectTimeSpinbox">
          <property name="value">
           <number>60</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="sliceLabel">
          <property name="text">
           <string>Slice time (seconds)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sliceTimeSpinbox">
          <property name="maximum">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Frame amount</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="frameAmountSpinbox">
          <property name="maximum">
           <number>9999999</number>
          </property>
          <property name="value">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="timeList">
          <property name="maximumSize">
           <size>
            <width>140</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="sourcesVLayout">
             <property name="spacing">
              <number>0</number>
             </property>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="refreshButton">
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Database Refresh</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="2">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="saveCSVFRAMEButton">
          <property name="text">
           <string>Save frame CSV</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveCSVEPISODEButton">
          <property name="text">
           <string>Save episode CSV</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="loadCSVButton">
          <property name="text">
           <string>Load CSV</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
